### DISCUSSION TITLE
Consider registering the Slint language in Linguist (adds syntax highlighting and language detection on GitHub)

### DISCUSSION THREAD

**Discussion by spenserblack** [slint-ui/slint#4531] [General]
Based on a [quick search](https://github.com/search?q=path%3A*.slint+NOT+is%3Afork+NOT+owner%3Aslint-ui&type=code&p=1), it looks like the Slint language (`.slint` files) has enough usage to meet the requirements to be added to [linguist](https://github.com/github-linguist/linguist). Linguist is the tool that GitHub uses to detect the languages used in a repository, and vendors the syntax highlighter for that language. Additionally, the data from linguist eventually makes its way over to [go-enry](https://github.com/go-enry/go-enry), which GitHub's search uses.

**Comment by ogoffart** [comment]
I look forward to add Slint to linguist, but according to https://github.com/github-linguist/linguist/issues/5756 , the threshold is 2000 files, and the search link you provide show only 1.6k files at the moment.
But we're getting there soon.

**Reply by spenserblack** [reply]
Oops, I misremembered the threshold :sweat_smile:
Thanks for finding the real threshold!

**Reply by spenserblack** [reply]
I happened to stumble on your PR (https://github.com/github-linguist/linguist/pull/6750), so I'm closing this discussion as resolved 🙂
